src/api-service/__app__/onefuzzlib/jobs.py

```diff
@@ -36,6 +40,34 @@ def search_expired(cls) -> List["Job"]:
             query={"state": JobState.available()}, raw_unchecked_filter=time_filter
         )
 
+    @classmethod
+    def stop_never_started_jobs(cls) -> None:
+        time_filter = "Timestamp lt datetime'%s'" % (
+            (datetime.utcnow() - JOB_NEVER_STARTED_DURATION).isoformat()
+        )
+        for job in cls.search(
+            query={
+                "state": [JobState.enabled],
+            },
+            raw_unchecked_filter=time_filter,
+        ):
+            # if the job has started, leave it be
+            if job.end_time is not None:
```

Comment:
can this be added to the db query?

Reply:
I got a failure when I tried in a fresh instance, as the field won't be created yet.

Reply:
I am ok to leave it as is but there is trick to check fields that might not exist we use it to [find outdated nodes](https://github.com/microsoft/onefuzz/blob/657af9722ceae3acc2e027f4181b3c62e96555fd/src/api-service/__app__/onefuzzlib/workers/nodes.py#L113)

Reply:
oh neat.  I'll investigate that.

Reply:
Updated following that example.